docs/fixers.rst

```diff
@@ -281,6 +281,16 @@ to specify the ``-f default`` or ``--fix=default`` option, e.g.::
 
     python-modernize -f default -f libmodernize.fixes.fix_open
 
+.. 2to3fixer:: classic_division
+
+   When a use of the division operator -- ``/`` -- is found, add
+   ``from __future__ import division`` and change the operator to ``//``.
+   This fixer is opt-in because some objects may override the ``__div__`` method
+   for a use other than division and thus would break when suddenly changed to
+   use a ``__floordiv__`` method instead.
```

Comment:
More fundamentally, floordiv is not equivalent to classic division even for built in types - e.g. `1.5/2 !== 1.5//2` in Python 2. Floordiv and classic division are only the same when both operands are integers.

Given how many users learn to make one or other of the numbers in a division a floating point number specifically to avoid integer division, I think the caveat should be clearer.